src/atoms/Photography/photography.module.css

```diff
@@ -0,0 +1,8 @@
+.photography {
+  text-align: center;
+  background-image: url("../../images/image_1.jpg");
+  background-repeat: no-repeat;
+  background-size: 776px 214px;
+  background-position: center center;
+  height: 200px;
```

Comment:
This value should be migrated to custom property in this same file.

Reply:
Use React component props?

Reply:
Done.